docs/compass/04-01-installation.md

```diff
@@ -390,9 +389,9 @@ To install the Compass and Runtime components on a single cluster, perform the f
 
 > **NOTE:** During the installation of Kyma, the installed version must match to the one in the [`KYMA_VERSION`](../../installation/resources/KYMA_VERSION) file in the specific Compass commit.
 
-You must have a Kyma installation with enabled Runtime Agent. For more information, see [Enable Kyma with Runtime Agent](https://github.com/kyma-project/kyma/blob/2.13.1/docs/04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md). Therefore, you must add the compass-runtime-agent module in the kyma-system namespace to the list of [minimal kyma components file](../../installation/resources/kyma/kyma-components-minimal.yaml).
+You must have a Kyma installation with enabled Runtime Agent. For more information, see [Enable Kyma with Runtime Agent](https://github.com/kyma-project/kyma/blob/2.13.1/docs/04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md). Therefore, you must add the compass-runtime-agent module in the kyma-system namespace to the list of [Minimal Kyma Components File](../../installation/resources/kyma/kyma-components-minimal.yaml).
```

Comment:
Therefore, you must add the compass-runtime-agent module in the kyma-system namespace to the list of the minimal Kyma components file. See [Minimal Kyma Components File](../../installation/resources/kyma/kyma-components-minimal.yaml).